LamestWebserver/LamestWebserver/Core/Logger.cs

```diff
@@ -28,6 +73,15 @@ public class Logger
         /// </summary>
         public ELoggingLevel MinimumLoggingLevel = DefaultMinimumLoggingLevel;
 
+        internal EOutputSource currentOutputSource = EOutputSource.File | EOutputSource.Console | EOutputSource.Custom;
+
+        internal string currentFilePath = "LWS" + DateTime.Now.ToFileTime().ToString() + ".log";
+
+        internal MultiStream multiStream = new MultiStream();
+
+        internal StreamWriter streamWriter;
```

Comment:
Are these `internal`s supposed to be `private` or `protected` or actually `internal`. I'd probably prefer `protected`.